packages/frontend/src/mobx-transient.ts

```diff
@@ -0,0 +1,124 @@
+import { BaseTransient, App } from '@h4bff/core';
+import { autorun, observable } from 'mobx';
+import { useLocalStore } from 'mobx-react-lite';
+import { useContextApp } from './app-context';
+import { useEffect } from 'react';
+import { ClassConstructor } from '@h4bff/core/build/internal';
```

Comment:
We shouldn't import from `build` folder. Can we export it regularly?

Reply:
removed import entirely